src/routes/user.route.js

```diff
@@ -0,0 +1,22 @@
+// create an express router in src/routes/user.route.js:
+const express = require('express')
+const { randomBytes } = require('crypto')
+
+const USERS = []
+const router = express.Router()
+
+router.put('/', (req, res) => {
+    const { firstName, lastName} = req.body
+    const user = {
+        id: randomBytes(8).toString('hex'),
+        firstName,
+        lastName,
+        createdAt: Date.now(),
```

Comment:
values of `createdAt` and `updatedAt` should be in ISO format instead of timestamp